--- configure.in.old	2009-10-03 10:49:28.000000000 -0500
+++ configure.in	2009-10-03 10:50:28.000000000 -0500
@@ -226,7 +226,7 @@
   AC_PROG_LN_S
 fi
 
-AC_CONFIG_FILES([Makefile src/Makefile igraph.pc igraph_Info.plist debian/Makefile doc/Makefile doc/book/Makefile interfaces/R/configure.in interfaces/R/Makefile interfaces/R/src/Makevars.win interfaces/R/DESCRIPTION interfaces/Makefile interfaces/shell/Makefile])
+AC_CONFIG_FILES([Makefile src/Makefile igraph.pc igraph_Info.plist doc/Makefile doc/book/Makefile interfaces/R/configure.in interfaces/R/Makefile interfaces/R/src/Makevars.win interfaces/R/DESCRIPTION interfaces/Makefile interfaces/shell/Makefile])
 AC_OUTPUT
 
 AC_MSG_RESULT([igraph successfully configured.])
